### PR TITLE
Added Specificity,responsiveness and common pitfalls section to ReadMe

### PR DESCRIPTION
**Pull request by Sandhya-Madhuri**
FIX: Added Responsiveness and Common Pitfalls

### CODE REVIEW

File: css-in-javascript/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Airbnb CSS-in-JavaScript Style Guide
 
-*A mostly reasonable approach to CSS-in-JavaScript*
+_A mostly reasonable approach to CSS-in-JavaScript_
```
Comment (ljharb) [inline css-in-javascript/README.md:3]: Please revert unrelated formatting changes, and don’t run prettier or autoformatting on a file if the repo isn’t using that formatted already.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: css-in-javascript/README.md (hunk 1 above)
Comment (Sandhya-Madhuri) [reply css-in-javascript/README.md:3]: Sorry for not checking above pr properly. I have closed this pr and raised another one. I have disabled the autoformatting. please check latest pr.
https://github.com/airbnb/javascript/pull/3053

Comment (ljharb) [reply css-in-javascript/README.md:3]: Please don’t make extra PRs - this one should have been updated.
